### ISSUE TITLE
Update Skate Park to include BMX, Roller Blading, and Scooter sports tags

### ISSUE THREAD

**Issue by sanecito** [openstreetmap/iD#7080]
Currently 'Skate Park' only includes 'skateboard' as a sport tag. Adding other sports that use skate parks like inline skating (roller blading), BMX, and Scooter converts it in iD to a 'Sport Pitch' which is less accurate.

While skate parks are most commonly associated to skateboards due to prominent figures like Tony Hawk, they do include these other sports. To add weight to this claim for those who haven't visited a skate park, [Wikipedia also lists these sports under 'skate park'](https://en.wikipedia.org/wiki/Skatepark).

**Comment by quincylvania** [comment]
Hi @sanecito, I agree this behavior isn't great, but the OSM tagging makes this tricky! Skateparks are tagged with the same base tag as soccer fields and tennis courts, `leisure=pitch`, meaning iD can only differentiate between different pitches via the `sport` tag. iD's presets cannot parse "one of multiple" tags or multiple values for the same tag, so we can't address the issue as-is.

I'd recommend solving this by switching the OSM tagging to something that better defines what the feature is, like `leisure=skatepark`, instead of simply what you can do there. This would be in line with a tag like `leisure=ice_rink` that accepts `sport` values but doesn't rely on them.

**Comment by sanecito** [comment]
Thanks for the quick, excellent feedback! I'll see what I can do this week to fix this upstream in OSM going off the proposal process documented in the OSM wiki to write and request review of such a `leisure=skatepark` tag

**Comment by sun-geo** [comment]
@sanecito just FYI
It seems there are some of such elements already available in the osm-db, see:
https://taginfo.openstreetmap.org/tags/leisure=skate_park
https://overpass-turbo.eu/s/OEI

**Comment by sanecito** [comment]
Hey @sun-geo thanks for highlighting exact usage counts (looks like about 150 across both skate_park and skatepark). Yes, in the course of researching leisure and skating tags in the OSM Wiki to draw up my proposal draft I found [this blurb](https://wiki.openstreetmap.org/wiki/Tag:sport%3Dskateboard):

> These combinations are not used in high numbers as of Sept 2018, but could be considered for skateparks:
> 
>     `leisure=skatepark`
>     `leisure=skate_park`

Having a standardized `leisure=skatepark` should help fix this duality as well as other issues I'll highlight in the rationale section in the next couple of hours when the page is created.

**Comment by sanecito** [comment]
Tagging proposal is now in RFC, will probably keep in RFC a week before moving to Voting phase
https://lists.openstreetmap.org/pipermail/tagging/2019-December/049619.html

**Comment by quincylvania** [comment]
@sanecito Thanks for taking up this proposal and moving it forward!

One note: I'm personally not sure adding a distinct `leisure=bike_park` is needed since they seem relatively uncommon and their form is nearly identical to skateparks. We could add a "Bike Park" preset in iD using `leisure=skatepark` + `sport=bmx` 🤷‍♂️

**Comment by bhousel** [comment]
> One note: I'm personally not sure adding a distinct `leisure=bike_park` is needed since they seem relatively uncommon and their form is nearly identical to skateparks. We could add a "Bike Park" preset in iD using `leisure=skatepark` + `sport=bmx` 🤷‍♂️

I would assume `leisure=bike_park` as a mountain biking park  - https://mtbparks.com/Bike-Parks/
They often double as ski mountains in off season, but many have dedicated stuff build up just for bikes.

**Comment by sanecito** [comment]
> One note: I'm personally not sure adding a distinct `leisure=bike_park` is needed since they seem relatively uncommon and their form is nearly identical to skateparks. We could add a "Bike Park" preset in iD using `leisure=skatepark` + `sport=bmx`

Yes, I'm not sure how many instances need to exist in order for it to be justifiable to get an approved tag. While the US has 3,500+ skateparks as of 2014, it looks like going off of [this list](https://www.twowheelingtots.com/kid-friendly-bike-park-pump-track-directory/) there are roughly 310 bike parks + the ones referenced in @bhousel's list.

The ones I had in mind were the ones in [Houston](https://www.usabmx.com/tracks/2002) and [Chandler](https://www.chandleraz.gov/explore/chandler-parks/guide/espee-park/bike-park?pageid=727) that are skatepark-like in nature, but the ones bhousel would also qualify as bike park. I couldn't find a Wikipedia-like definition for Bike Park like skatepark, but I found [this](https://www.flowformramps.com/about/what-bike-park) which encompasses both skatepark types and mountain bike trail types although it has more emphasis on MTB elements:

> A bike park is a piece of land specifically designed for off-road or mountain biking. Bike parks can be built in small urban areas areas, public lands or integrated with trail systems. A bike park may include skills courses, pump tracks, jumps, trails and features. With many design and budget options available, a bike park is an achievable project for all communities.

I can re-add the BMX only example of `leisure=skatepark` + `sport=bmx` to the proposal, but I'd prefer tagging consensus on the mailing list / proposal wiki comments. I feel like recommending `leisure=bike_park` (and not creating an approved leisure=bike_park given the low count) would have the least amount of contention, but I'm not a tagging list regular, so don't have a good read. I'll have to see how those MTBParks are tagged to see if it sparks any ideas. I imagine they're a combination of trail and biking sport tags though.

**Comment by matkoniecz** [comment]
> Yes, I'm not sure how many instances need to exist in order for it to be justifiable to get an approved tag.

"approved" tag is tag that went through https://wiki.openstreetmap.org/wiki/Proposal_process with a successful vote.

**Comment by sanecito** [comment]
> 
> 
> > Yes, I'm not sure how many instances need to exist in order for it to be justifiable to get an approved tag.
> 
> "approved" tag is tag that went through https://wiki.openstreetmap.org/wiki/Proposal_process with a successful vote. Usage count is a separate metric.

Correct, but as a first time tag submitter, I wasn't sure if there's some soft, undocumented number in the wiki the people who typically vote prefer. For example earlier in the thread by quincylvania:

> I'm personally not sure adding a distinct leisure=bike_park is needed since they **seem relatively uncommon** and their form is nearly identical to skateparks

Basically, I want to figure out the best resolution to this issue to possibly avoid people voting against proposal because they for example don't agree on the side rationale of creating a `leisure=bike_park` and prefer some other method. I should hopefully have enough examples in the next 12 hours to propose how to deal with bike parks' special case to ask people to vote one way or another in the mailing list thread and adjust the wiki proposal accordingly. Then the proposal phase itself can move to voting.

**Comment by tastrax** [comment]
Hi Folks,

 

Bike parks are becoming increasingly common in Australia where they are reviving many small towns

 

https://www.maydenabikepark.com/ - https://www.openstreetmap.org/#map=18/-42.75551/146.62833

https://www.ridebluederby.com.au/ - https://www.openstreetmap.org/#map=17/-41.14896/147.80255

 

The bike trail network covers quite a large area, often covering both private and reserved land. They are much larger than “skate parks” which are generally small and in urban playground settings

 

Cheers - Phil

**Comment by Adamant36** [comment]
> Bike parks are becoming increasingly common in Australia where they are reviving many small towns. The bike trail network covers quite a large area, often covering both private and reserved land. They are much larger than “skate parks” which are generally small and in urban playground settings

I wouldn't consider the complete trail systems as part of the bike park. There's already good tags for denoting biking trails anyway.

**Comment by quincylvania** [comment]
Hmm I had only considered urban bike parks that look essentially like skateparks. A tag like `leisure=bike_park` makes more sense to me when it covers mountain bike parks as well, since those are quite different from skateparks. Distinct tags for uncommon features is okay if they can't fit cleanly under existing tags.

**Comment by bhousel** [comment]
> Distinct tags for uncommon features is okay if they can't fit cleanly under existing tags.

Yes, this is why I still think it would be most convenient if people would be ok with just having their new tag fit under the existing `leisure=pitch`+`sport=*` scheme.

**Comment by quincylvania** [comment]
Tagging which could thread the needle is `leisure=pitch` + `pitch=skatepark` + `sport=*`. This would define the specific physical form while being backwards-compatible.

**Comment by sanecito** [comment]
@quincylvania Is there an existing tag preset, wiki entry, etc that makes use of `pitch=*`? If that's the best path forward, I'd need to figure out how to re-word the existing proposal draft suggested actions, and am currently not sure how. Would it just be creating a `pitch=*` tag, with the only valid tag value being `pitch=skatepark` for now?

**Comment by quincylvania** [comment]
@sanecito Looks like the [`pitch`](https://taginfo.openstreetmap.org/keys/pitch#values) key mostly contains `sport` values, probably by accident. There doesn't appear to be any "approved" values.

I think it's okay to narrowly propose this one tag under `pitch=` and leave additional values for future proposals, if needed.

**Comment by TheAdventurer64** [comment]
@sanecito Any progress on the proposal?

**Comment by sanecito** [comment]
@TheAdventurer64 

Thanks for the followup; took bereavement shortly after the prior conversation and then COVID hit and this fell between the cracks. Will target modifying the proposal within the next two weeks to Quincy's suggestion of creating a `pitch=*` and `pitch=skatepark` as well as updating the [leisure skatepark example](https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dpitch#Skatepark) to reflect the new potential `pitch=skatepark`
